review.md

```diff
@@ -0,0 +1,92 @@
+# Code Review principles and tips
+
+## Why do we code review?
+
+1. Code reviews are essential to improve the level of our code, and to prevent bugs. 
+
+2. A great and fast way for both the reviewer and the code owner to share ideas,  
```

Comment:
extra space here at the end is breaking the line on the rendered text as well
```suggestion
2. A great and fast way for both the reviewer and the code owner to share ideas, 
```

Reply:
fixed